frontend/packages/client/src/hooks/useCommunityUsersMutation.js

```diff
@@ -80,11 +80,11 @@ export default function useCommunityUsersMutation({ communityId } = {}) {
   const addCommunityUsers = useCallback(
     async ({ userType, addrs, body }) => {
       return new Promise((resolve, reject) => {
-        addrs.forEach((addrToRemove) => {
```

Comment:
change in the name only

Reply:
Nice - saw that when I was looking at this issue.